test/helpers/utils.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"fmt"
+	"os"
+)
+
+// CreateDir creates dir if does not exist.
+// The created dir will have the permission bits as 0777, which means everyone can read/write/execute it.
+func CreateDir(dirPath string) error {
+	return CreateDirWithFileMode(dirPath, 0777)
+}
+
+// CreateDirWithFileMode creates dir if does not exist.
+// The created dir will have the permission bits as perm, which is the standard Unix rwxrwxrwx permissions.
+func CreateDirWithFileMode(dirPath string, perm os.FileMode) error {
```

Comment:
Can we keep these in an existing file instead of creating a new one?

Reply:
I also prefer not to create a new file, but seems there is no appropriate file to put it in.
Maybe we can rename the `data.go` file to `utils.go`? Or any other suggestions?

Reply:
Based on your other reply, I think this is the wrong repo for such helper function.

Reply:
Are you suggesting to put it under `test-infra/shared`? Seems we'll also need to create a new file if doing so, since all existing folders are individual components. Is it fine if I create a file `test-infra/shared/common/common.go` for holding this function?

Reply:
I think so. @srinivashegde86 objections?

Reply:
nope. sg.